src/WebJobs.Script/Http/ScriptInvocationRequestTransformer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Yarp.ReverseProxy.Forwarder;
+
+public class ScriptInvocationRequestTransformer : HttpTransformer
+{
+    public override async ValueTask TransformRequestAsync(HttpContext httpContext, HttpRequestMessage proxyRequest, string destinationPrefix, CancellationToken cancellationToken)
+    {
+        await base.TransformRequestAsync(httpContext, proxyRequest, destinationPrefix, cancellationToken);
```

Comment:
Discussed this offline, but I'm concerned this will not maintain the default behavior we previously had. 

Your best course of action is  here would likely be to call `HttpTransformer.Default.TransformRequestAsync` instead, then add our custom logic on top.